src/pudl/output/eia930.py

```diff
@@ -8,16 +8,42 @@
     impute_timeseries_asset_factory,
 )
 
-BA_TIMEZONE_MAP = {
-    "CISO": "America/Los_Angeles",
-    "ERCO": "America/Chicago",
-    "ISNE": "America/New_York",
-    "MISO": "America/Chicago",
-    "NYIS": "America/New_York",
-    "PJM": "America/New_York",
-    "PNM": "America/Denver",
-    "SWPP": "America/Chicago",
-}
+
+def _add_timezone(
```

Comment:
Switched to getting timezones from the `core_eia__codes_balancing_authorities` table.